api-php/samples/makefile

```diff
@@ -0,0 +1,17 @@
+# Variables
+UI_WEB_DIR=../../ui-web/samples
```

Comment:
Could we use a `build-samples.ps1` file or `run-samples.ps1` instead of make? Now we would have two ways.